/*
 * Created by shadowsparky in 2019
 *
 */

package ru.shadowsparky.client.exceptions

/**
 * Исключение, возникающее когда пользователь пытается запустить трансляцию, не завершив предыдущую.
 */
class ProjectionAlreadyStartedException : Exception()